### DISCUSSION TITLE
All the success messages and error message and stuff contained in "form"[Bug]

### DISCUSSION THREAD

**Discussion by bapbassa** [webstudio-is/webstudio-community#153] [Q&A] [answered]
The video and explanations about creating a form in the documentation is not valid anymore.

When I add a form element all the messages and also the "state" to choose in the settings disappear.

How do I do to set up my form submition ?

Thanks in advance !

**Comment by kof** [comment, marked as answer]
Form component you often see int he videos is now called Webhook Form

**Reply by bapbassa** [reply]
Thank you very much !!
